src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailService.java

```diff
@@ -47,6 +48,13 @@ public EmailService(
         this.dateTimeSupplier = dateTimeSupplier;
     }
 
+    /**
```

Comment:
Rather than add this in, use ReflectionTestUtils.setField() method in your test to set the subject.